### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by arjxn-py**
Discussed in #405 
Changes are auto-generated with `jlpm install` locally

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1224,32 +1224,32 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@jupyterlab/apputils@npm:^4.0.0":
-  version: 4.3.5
-  resolution: "@jupyterlab/apputils@npm:4.3.5"
+"@jupyterlab/apputils@npm:~4.0.0":
+  version: 4.0.0
```
Comment (martinRenou) [inline yarn.lock:1228]: I'm surprised this get downgraded.

**Review by martinRenou — COMMENTED** [review]
I'm surprised by one of the changes, also when I try `jlpm install` locally it does not seem to change the lock file for me. I'm not sure what's different in your setup

File: yarn.lock (hunk 1 above)
Comment (arjxn-py) [reply yarn.lock:1228]: So am I, it doesn't look right to me too.

Comment (arjxn-py) [reply yarn.lock:1228]: I think I shall try troubleshooting this issue for sometime locally first and then get back

**Comment by arjxn-py** [comment]
Oh, it works. 
I guess it was an issue locally while switching branches, seems like `package.json` was incorrectly overwritten. Apologies.

**Comment by martinRenou** [comment]
No worries, thanks for checking!

**Pull request closed without merging**